bluebottle/funding/models.py

```diff
@@ -572,6 +572,12 @@ class PayoutAccount(ValidatedModelMixin, PolymorphicModel, TransitionsMixin):
     updated = models.DateTimeField(auto_now=True)
     reviewed = models.BooleanField(default=False)
 
+    @property
+    def funding(self):
+        for account in self.external_accounts.all():
+            for funding in account.funding_set.all():
+                return funding
```

Comment:
or `return account.external_accounts.first().funding_set.first()`?

Reply:
That would work most of the time. But not if there would be accounts without fundings. In theory that could happen.